docs/tutorials/usage.md

```diff
@@ -37,12 +49,10 @@ from agrag.agrag import AutoGluonRAG
 def ag_rag():
     agrag = AutoGluonRAG(
         preset_quality="medium_quality", # or path to config file
-        web_urls=["https://auto.gluon.ai/stable/index.html"], # List of URLs to use for RAG
-        base_urls=["https://auto.gluon.ai/stable/"], # List of base URLs to use when processing web     
-                                                     # URLs. Only Web URLs that stem from a base URL 
-                                                     # will be processed.
-        parse_urls_recursive=True, # Whether to recursively parse all URLs from the provided web url list
-        data_dir="s3://autogluon-rag-github-dev/autogluon_docs/" # Directory containing files to use for RAG
+        web_urls=["https://auto.gluon.ai/stable/index.html"],
+        base_urls=["https://auto.gluon.ai/stable/"],
+        parse_urls_recursive=True,
+        data_dir="s3://autogluon-rag-github-dev/autogluon_docs/"
```

Comment:
Looks like external user do not have access to this s3 bucket.

Reply:
Yes but this function won't be called now if the user just does `agrag -h` as I fixed that part of the code in `agrag.py`.

Reply:
Maybe we should raise a `ValueError` if the user does not provide `data_dir`

Reply:
An error is raised in the data processing module, but I don't think there is a point in setting a default in `main.py`